src/sql/azdata.proposed.d.ts

```diff
@@ -640,13 +644,19 @@ declare module 'azdata' {
 
 		export type DialogWidth = 'narrow' | 'medium' | 'wide' | number;
 
+		export type DialogStyle = 'normal' | 'flyout' | 'callout';
+
+		export type DialogPosition = 'left' | 'right' | 'above' | 'below' | undefined;
```

Comment:
do we really need this? it would be fine as long as it shows up next to the anchor element, right?

Reply:
I wanted to build in some flexibility. At the very least, we need **left** and **below**.

Reply:
Understood. I'll remove 'right' and 'above'.